activerecord/lib/active_record/railties/query_comments.rb

```diff
@@ -0,0 +1,64 @@
+module ActiveRecord
+  module Railties # :nodoc:
+    module QueryComments #:nodoc:
+      module ActionController
+        extend ActiveSupport::Concern
+        included do
+          # TODO make this optional based on config, default enabled
```

Comment:
Let's either delete this comment before publishing to rails/rails or implement this.

Reply:
I've been caught in a loop of load ordering and location appropriateness indecision over this.. I think what I've got in here makes sense now.